docs/status.rst

```diff
@@ -0,0 +1,83 @@
+Status Report
+=============
+
+.. note:: This page will be updated as development progresses. If you have any questions or something appears out of date please feel free to :doc:`contact us <support>`.
+
+Development wraps around the three pillars identified with the help of CERN physicists in the `5th CAP meeting <https://github.com/cernanalysispreservation/analysis-preservation.cern.ch/wiki/Fifth-CAP-meeting>`_:
```

Comment:
I think this should link to https://github.com/cernanalysispreservation/analysispreservation.cern.ch/wiki/Overview